src/plugins/vis_type_timeseries/common/agg_lookup.ts

```diff
@@ -1,122 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import { omit, pick, includes } from 'lodash';
-import { i18n } from '@kbn/i18n';
-import { Metric } from './types';
-
-export const lookup: Record<string, string> = {
-  count: i18n.translate('visTypeTimeseries.aggLookup.countLabel', { defaultMessage: 'Count' }),
-  calculation: i18n.translate('visTypeTimeseries.aggLookup.calculationLabel', {
-    defaultMessage: 'Calculation',
-  }),
-  std_deviation: i18n.translate('visTypeTimeseries.aggLookup.deviationLabel', {
-    defaultMessage: 'Std. Deviation',
-  }),
-  variance: i18n.translate('visTypeTimeseries.aggLookup.varianceLabel', {
-    defaultMessage: 'Variance',
-  }),
-  sum_of_squares: i18n.translate('visTypeTimeseries.aggLookup.sumOfSqLabel', {
-    defaultMessage: 'Sum of Sq.',
```

Comment:
`agg_select.tsx` and `agg_lookup.ts` had 2 different labels so after removing duplications '... Sq.' was replaced with '... Squares'

Reply:
👍 We want the aggregation name to match its generated label so I think it's okay. Thank you!